### ISSUE TITLE
autoreduction_processor_daemon consuming 100% CPU on prod

### ISSUE THREAD

**Issue by DavidFair** [ISISScientificComputing/autoreduce#109]
Investigate why the autoreduction_processor_daemon is consuming a CPU core continuously on the reduction queue for production.

**Comment by Anders-Markvardsen** [comment]
Looks find on production node autoreduce. Tested with `ps -aux | gre autoreduction`, get the PID of autoreduction_processor_daemon and use top, where this daemon was showing OK because consuming between 0 and 1% of CPU. 

Possibly this PID was confused with PIDs of actual autoreduction jobs

**Comment by Anders-Markvardsen** [comment]
Tested on from scratch installed cloud Autoreduction instance. It was observed that running the processors QueueProcessor and autoreduction_processor_daemon then autoreduction_processor_daemon use zero CPU, and QueueProcessor uses 100%, i.e. it grabs hold of one thread and polls ActiveMQ, which is fine.

On production see this behaviour also QueueProcessor grabs hold of a single thread but not more. When autoreduction jobs are run these grabs multiplete available threads, in a test this was listed as ~600% with Top.

In summary and agreeing with Elliot this behavior is acceptable and no need to take any further action
